src/Inc/Menus.php

```diff
@@ -0,0 +1,61 @@
+<?php namespace Leean\Endpoints\Inc;
+
+/**
+ * Class Menus
+ *
+ * @package Leean\Endpoints\Inc
+ */
+class Menus
+{
+	/**
+	 * Returns an array of menu locations, with the assigned menu in each.
+
+	 * @return array
+	 */
+	public static function get() {
```

Comment:
I think this should be named `get_locations` if it is what the function returns.